### DISCUSSION TITLE
Enivronment Warning !

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#832] [General Q&A] [answered]
![image](https://user-images.githubusercontent.com/88183889/134291171-5d981f91-76b4-44d7-bbbb-725e90ad6cbe.png)

How can I change the Environment ?  to get rid off this error..
Now currently Environment is production ..I need to change it as Development ?
Give me idea which file is responsible for this ? because I have tried in server.py file nothing impacting this warning

**Comment by erogol** [comment, marked as answer]
It is not an error it's a warning. The server is not intended for deployment only for testing out models.

**Reply by ghost** [reply]
Debug mode : off How can we ON the debug mode ?
